src/main/java/frc/commands/Lift.java

```diff
@@ -0,0 +1,47 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.OI;
+import frc.robot.Robot;
+
+public class Lift extends Command {
+  public Lift() {
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    Robot.mast.lift((OI.xbox.getRawAxis(1)) * 0.6);
```

Comment:
prob change xbox to aux so its on the right controller

```suggestion
    Robot.mast.lift((OI.aux.getRawAxis(1)) * 0.6);
```